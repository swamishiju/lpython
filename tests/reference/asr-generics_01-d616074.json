{
    "basename": "asr-generics_01-d616074",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/generics_01.py",
    "infile_hash": "3062320cca6688cea4b76b4c18dbaf5d1dcc1de193459ee598e84935",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-generics_01-d616074.stderr",
    "stderr_hash": "7aac97cbff36f55dbe879fc690d9ea0d40262a9286c36b808f882e4e",
    "returncode": 2
}