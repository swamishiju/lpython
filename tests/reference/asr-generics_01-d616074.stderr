semantic error: Type mismatch in procedure call; the types must be compatible
  --> tests/../integration_tests/generics_01.py:18:9
   |
18 | print(f(1, 2, add=add_integer))
   |         ^ type mismatch (passed argument type is i32 but required type is T)
   |
15 | def f(x: T, y: T, **kwargs) -> T:
   |          ^ type mismatch (passed argument type is i32 but required type is T)
